eng/testing/performance/performance-setup.ps1

```diff
@@ -152,7 +152,12 @@ if ($iOSMono) {
     {
         mkdir $WorkItemDirectory
     }
-    Copy-Item -path "$SourceDirectory\iosHelloWorld\nollvm" $PayloadDirectory\iosHelloWorld\nollvm -Recurse
+    if($iOSLlvmBuild) {
+        Copy-Item -path "$SourceDirectory\iosHelloWorld\nollvm" $PayloadDirectory\iosHelloWorld\llvm -Recurse
+    } else {
+        Copy-Item -path "$SourceDirectory\iosHelloWorld\llvm" $PayloadDirectory\iosHelloWorld\nollvm -Recurse
+    }
+
```

Comment:
I guess this prevents it from going to down to Helix machines, but we still download the artifacts for both as part of the AzDO job.